### PR TITLE
Validate ratechecker load from external file

### PR DESCRIPTION
**Pull request by chosak**
This PR modifies how the `load_daily_data` management command validates the new data that it loads. Instead of using a set of hardcoded scenarios defined in a Python module, it instead reads in an external file defining the scenarios to use.

The external file must be in [JSONL format](http://jsonlines.org/) (file where each line is JSON formatted) and consist of a list of API parameters encoded as key-value pairs. The validation logic has been pulled into a new `ratechecker/validation.py` module but has not been changed. As before,
the expected results are loaded from an XML file in the data archive.

This commit makes one slight change to how [API parameters](http://cfpb.github.io/owning-a-home-api/api/) are validated. The API always requires a `loan_amount` parameter which is the amount of the loan being requested. It can take a `price` parameter which is the cost of the house being purchased or refinanced; if this is provided, then an `ltv` parameter is computed that represents the loan-to-value ratio. The API validation scenarios, however, include this `ltv` parameter directly, and don't include `price`. To make the API more straightforward and consistent, this commit includes a minor change to how parameters are validated that require exactly one of `price` or `ltv`. New unit tests have been added to verify this change, and it won't have any impact on how the API behaves on consumerfinance.gov.

This commit explicitly does not attempt to improve the overall load process whereby data is copied to temp tables and then copied back on failure. This work is scheduled for a future task and so was not
included here so as to limit scope of this particular change.

cc: @higs4281

Ping me if you need to get some sample data to test this against, or the validation scenarios, as these are no longer available on public GitHub.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 All notable changes to this project will be documented in this file.
 We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
+## Unreleased
+- Load validation scenarios from external file instead of Python module.
```
Comment (willbarton) [inline CHANGELOG.md]: 🎉

Comment (willbarton) [inline CHANGELOG.md]: Unrelated: would it be worth github-changelog'ing this repo too?

File: ratechecker/management/commands/load_daily_data.py [hunk 2]
```diff
@@ -1,117 +1,129 @@
+from __future__ import print_function
```
Comment (willbarton) [inline ratechecker/management/commands/load_daily_data.py]: 🎉

File: CHANGELOG.md (hunk 1 above)
Comment (chosak) [inline CHANGELOG.md]: Thanks for the review. That's not a bad idea, and more generally we could use that approach for more of the cfgov-refresh satellite apps. I do think that CHANGELOGs continue to make sense for repos where versioning is important, primarily those where there might be multiple (non-CFPB) users of the library.

**Pull request merged**